src/elements.jl

```diff
@@ -96,10 +124,20 @@ function local_potential_fourier(el::ElementPsp, q::T) where {T <: Real}
 end
 
 function local_potential_real(el::ElementPsp, r::Real)
-    # Use local part of pseudopotential defined in Element object
     return eval_psp_local_real(el.psp, r)
 end
 
+function valence_charge_density_fourier(el::ElementPsp, q::T) where {T <: Real}
+    if has_rho_valence(el.psp)
+        eval_psp_rho_valence_fourier(el.psp, q)
+    else
+        gaussian_valence_charge_density_fourier(el, q)
+    end
+end
+
+function core_charge_density_fourier(el::ElementPsp, q::T) where {T <: Real}
+    eval_psp_rho_core_fourier(el.psp, q)
+end
```

Comment:
I wrote it like this because the family of functions that operate on `NormConservingPsp`s are all prefixed by `eval_psp_` and then generally use a more compact name for the quantity: e.g. `eval_psp_local_fourier(::NormConservingPsp)` vs `local_potential_fourier(::Element)`. So, I think that this is actually fairly consistent with the interfaces for `Element` and `NormConservingPsp`.

Reply:
I agree, but what I meant was actually to give up on the `eval_psp_local_fourier` functions completely and instead use `local_potential_fourier(::NormConservingPsp)`.